po/es.po

```diff
@@ -11128,10 +11129,11 @@ msgid ""
 "contrast\n"
 "'averaged guided filter' is a geometric mean of both methods"
 msgstr ""
-"'no' afecta el contraste global y local (seguro si solo agrega contraste)\n"
-"'filtro guiado' solo afecta el contraste global tratando de preservar el "
+"No: afecta el contraste global y local (es seguro si solo se añade "
+"contraste)\n"
+"Filtro guiado: solo afecta el contraste global tratando de preservar el "
 "contraste local\n"
-"'filtro guiado promediado' es una media geométrica de ambos métodos"
+"Filtro guiado promediado: es una media geométrica de ambos métodos"
```

Comment:
Same comment about quotes and colon as before

Reply:
Same answer ;-P